### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by IgnacioColuccio**
### Summary
- Implemented all features detailed for the tutorial blog project in http://guides.rubyonrails.org/getting_started.html.
- Changed sqlite3 to postgresql, and erb to slim.

### Trello Card
https://trello.com/c/n1LLvNdk/23-crear-blog-base

### Test Plan
(when prompted, username is "dhh", password is "secret"
1. Go to /articles
2. Create a new article. It should prompt for credentials.
3. Add new comments. 
4. Comments should be visible on the article page.
5. Destroy a comment. It should display a confirm window and then prompt for credentials (inputting wrong credentials should prevent the comment from being deleted).
6. Go back to /articles.
7. Destroy the article. It should display a confirm window and then prompt for credentials.
8. Comment should no longer appear on the listing page.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,6 +5,7 @@
 gem 'rails', '4.2.4'
 # Use postgresql as the database for Active Record
 gem 'pg'
+#gem 'sqlite3'
```
Comment (Dinuuu) [inline Gemfile]: Do not leave gems commented in the gemfile

File: app/assets/javascripts/articles.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Dinuuu) [inline app/assets/javascripts/articles.coffee]: Delete unnecessary files

File: app/assets/javascripts/comments.coffee [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Dinuuu) [inline app/assets/javascripts/comments.coffee]: idem

File: app/assets/javascripts/welcome.coffee [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Dinuuu) [inline app/assets/javascripts/welcome.coffee]: idem

File: app/assets/stylesheets/articles.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the articles controller here.
```
Comment (Dinuuu) [inline app/assets/stylesheets/articles.scss]: idem

File: app/assets/stylesheets/comments.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Comments controller here.
```
Comment (Dinuuu) [inline app/assets/stylesheets/comments.scss]: idem

File: app/assets/stylesheets/welcome.scss [hunk 7]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the welcome controller here.
```
Comment (Dinuuu) [inline app/assets/stylesheets/welcome.scss]: idem

File: app/helpers/articles_helper.rb [hunk 8]
```diff
@@ -0,0 +1,2 @@
+module ArticlesHelper
```
Comment (Dinuuu) [inline app/helpers/articles_helper.rb]: idem

File: app/helpers/comments_helper.rb [hunk 9]
```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
```
Comment (Dinuuu) [inline app/helpers/comments_helper.rb]: idem

File: app/helpers/welcome_helper.rb [hunk 10]
```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
```
Comment (Dinuuu) [inline app/helpers/welcome_helper.rb]: idem

File: config/database.yml [hunk 11]
```diff
@@ -20,6 +20,8 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: 5
+  username: ignaciocoluccio
```
Comment (Dinuuu) [inline config/database.yml]: We tend to use the name of the project as username and password for development stages, you don't have to change this, but it was for you to know it :)

File: test/controllers/articles_controller_test.rb [hunk 12]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (Dinuuu) [inline test/controllers/articles_controller_test.rb]: As we talked before when the time comes you would have to use rspec gem that uses the spec folder to store the tests so we can erase this ones

File: test/controllers/comments_controller_test.rb [hunk 13]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
```
Comment (Dinuuu) [inline test/controllers/comments_controller_test.rb]: idem

File: test/controllers/welcome_controller_test.rb [hunk 14]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
```
Comment (Dinuuu) [inline test/controllers/welcome_controller_test.rb]: idem

File: test/models/article_test.rb [hunk 15]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
```
Comment (Dinuuu) [inline test/models/article_test.rb]: idem

File: test/models/comment_test.rb [hunk 16]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CommentTest < ActiveSupport::TestCase
```
Comment (Dinuuu) [inline test/models/comment_test.rb]: idem

**Comment by IgnacioColuccio** [comment]
Removed all unneeded files

**Comment by Dinuuu** [comment]
Everything is ok, squash those last 3 commits into one named `Pull Request Fixes` and merge it :+1:

**Pull request merged**